addons/rearm/functions/fnc_rearmEntireVehicleSuccessLocal.sqf

```diff
@@ -17,60 +17,42 @@
  */
 #include "script_component.hpp"
 
-params [["_truck", objNull, [objNull]], ["_vehicle", objNull, [objNull]], ["_turretPath", [], [[]]]];
-TRACE_3("rearmEntireVehicleSuccessLocal",_truck,_vehicle,_turretPath);
+params ["_truck", "_vehicle", "_turretPath"];
+TRACE_3("rearmEntireVehicleSuccessLocal", _truck, _vehicle, _turretPath);
```

Comment:
So is that not wanted? And what would it cause in other types of macro, like say
`EFUNC(interact_menu, createAction)`?